docs/docs/testing/e2e/app-template.md

```diff
@@ -43,4 +43,54 @@ This endpoint `DELETES` every row from every table _excluding_ the "core" tables
 
 #### Why do we have it?
 
-This lets us wipe the entire test instance _if_ we need to. DTS does technically does this already for us. But nonetheless, its useful.
+This lets us wipe the entire test instance _if_ we need to. DTS does technically
+does this already for us. But nonetheless, its useful.
+
+### Config
+
+found at `template/src/api/config`
+
+#### Rate Limit
+
+##### Usage
+
+```ts
+async function toggleRateLimiting(page, enabled = true) {
+  await page.request.fetch('/api/config/ratelimit/enable', {
+    method: 'POST',
+    data: { value: enabled },
+  });
+}
+```
+
+##### What does it do?
+
+This endpoint can be used to enable or disable to rate limitting middleware in
```

Comment:
not sure if it's just me but 
`can be used to enable or disable to rate limitting middleware`
sounds a bit strange to me, wdyt?